packages/api/lambdas/message-consumer.js

```diff
@@ -199,9 +199,7 @@ function processRecord(record, fromSNS) {
     } catch (err) {
       log.error('Caught error parsing JSON:');
       log.error(err);
-      if (fromSNS) {
-        return handleProcessRecordError(err, record, isKinesisRetry, fromSNS);
-      }
+      return handleProcessRecordError(err, record, isKinesisRetry, fromSNS);
```

Comment:
If we are in this `else` AND `fromSNS` is `true`, then it is already a kinesisRetry, and `handleProcessRecordError` will throw an error regardless. 

Really the only thing this is doing is keeping bad JSON parses from being retried so that they never throw an error. I'm not sure that makes sense. I don't know why I added this a year ago in the first place. 

I'm not entirely convinced that it makes sense to call `handleProcessRecordError` in this block at all, but the `if` just made no logical sense to me at all at this point.